test/parallel/test-net-client-bind-twice.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+// This tests that net.connect() from a used local port throws EADDRINUSE.
+
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+const server1 = net.createServer(common.mustNotCall());
+server1.listen(common.PORT, '127.0.0.1', common.mustCall(() => {
```

Comment:
Can we avoid using `common.PORT`? If not I think it's better to move this test to sequential.